test/integration/I18n_test.rb

```diff
@@ -190,23 +190,13 @@ class I18nTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test 'should choose i18n for user/_form + user/new' do
```

Comment:
@jywarren I pushed this change from my  laptop because we don't need this test anymore. We are not getting redirected to the signup page on clicking the signup button. As Jonathan was having trouble with tests. I pushed it. I hope pushing in other's pr to help them is okay with you @jywarren.